velox/type/DecimalUtil.cpp

```diff
@@ -54,41 +54,42 @@ const int128_t DecimalUtil::kPowersOfTen[]{
     1,
     10,
     100,
-    1000,
```

Comment:
I made this change to enhance the readability. But this is not required for this PR.